### ISSUE TITLE
big reference image in report points to wrong folder

### ISSUE THREAD

**Issue by zigotica** [garris/BackstopJS#902]
hi,

being on the report page, reference image preview shows correctly, while on click, it opens the test image

this was pointed out in https://github.com/garris/BackstopJS/issues/850#issuecomment-432704234 but I think it should have its own issue ID

**Comment by zigotica** [comment]
Yup, sorry let me rephrase @garris  

In the report page, the reference images preview look fine. But if you click on a reference image preview you get the "big" test image, not the "big" reference one. 

I found this on #850 where we have a reference and test that look different but not detected as such, causing the diffImage to not exist. In that situation, apparently, the system tries to build the slider comparator (but does not show the vertical red line), causing the overlay to only show the test image.

**Comment by screendriver** [comment]
Same here: reference image and shows the correct image. On the right side it shows the correct test image **but** it doesn't show any failing test (they should fail because they are different) and when I click the reference image to enlarge it, it shows me the test image instead of the reference image.

**Comment by ledenis** [comment]
This bug is still there. I will make a PR when I have time.
